buildifier/buildifier.go

```diff
@@ -24,11 +24,17 @@ import (
 	"io/ioutil"
 	"os"
 	"runtime"
-	"sort"
 	"strings"
+	"sync"
 
-	build "github.com/bazelbuild/buildifier/core"
-	"github.com/bazelbuild/buildifier/differ"
+	core "github.com/bazelbuild/buildifier/core"
```

Comment:
On Tue, Jul 19, 2016 at 11:46 AM, Andy Hochhaus <notifications@github.com>
wrote:

> In buildifier/buildifier.go
> <https://github.com/bazelbuild/buildifier/pull/6#discussion_r71365777>:
>
> >
> > -	build "github.com/bazelbuild/buildifier/core"
> > -	"github.com/bazelbuild/buildifier/differ"
> > +	core "github.com/bazelbuild/buildifier/core"
>
> Why explicitly name this "core"?
>

As a side note -- please don't rename that  the 'core -> build' renaming is
important to keep for pulling sources back to Google.
I will add a comment to that effect.


> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/bazelbuild/buildifier/pull/6/files/fff89537f62dd2868d478a4b9d512b23826994ec..6e9312af29eb5a6745707967feb7f04032d9d27b#r71365777>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ALF-0tYvLjjzk0Vj0z5DrXz6fvYZ6dlBks5qXPFNgaJpZM4JNFI0>
> .
>